### PR TITLE
Update mkFit to 2.0.1

### PR DESCRIPTION
**Pull request by makortel**
This PR updates mkFit to 2.0.1, which itself is a code-wise small update to have the same version whose performance will be presented on TRK POG meeting next Monday (Sep 9).

FYI @slava77 @mtosi @JanFSchulte

### CODE REVIEW

**Comment by makortel** [comment]
@cmsbuild, please test

File: mkfit-toolfile.spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### RPM external mkfit-toolfile 2.0.0
+### RPM external mkfit-toolfile 2.0.1
```
Comment (smuzaffar) [inline mkfit-toolfile.spec]: toolfile version is independent of tool itself. We only change it when the tool definition changes (e.g. add a new libdir, include or lib), So this change is not needed.

Comment (makortel) [reply mkfit-toolfile.spec]: Thanks, I restored the version.

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Comment by fabiocos** [comment]
+1

**Pull request merged**